data/printing/config.yaml

```diff
@@ -239,26 +269,39 @@ layouts:
               nbColumns: 1
               items:
                 - !text
-                  #width: 420
                   text: '(c) example.com'
                   fontEncoding: Cp1252
                   fontSize: 7
                   align: justified
                   vertAlign: top
                   spacingAfter: 2
                 - !text
-                  #width: 420
+                  text: '${author}'
+                  fontEncoding: Cp1252
+                  fontSize: 5
+                  align: left
+                  vertAlign: bottom
+                - !text
                   text: '${now MM.dd.yyyy}'
```

Comment:
all date should include also time 

![image](https://github.com/user-attachments/assets/48e62230-d5aa-454a-8ea0-e3301f8e9642)